src/box/wal_ext.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#pragma once
+
+#include "trivia/config.h"
+#include "say.h"
+
+#if defined(ENABLE_WAL_EXT)
+# include "wal_ext_impl.h"
+#else /* !defined(ENABLE_WAL_EXT) */
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+/** No-op in OS version. */
+static inline void
+wal_ext_free(void)
+{
+}
+
+struct wal_ext;
+struct space;
+struct txn_stmt;
+struct request;
+
+/** No-op in OS version. */
+static inline void
+wal_ext_space_process(struct space *space, struct txn_stmt *stmt,
+		      struct request *request)
```

Comment:
Please add comments to all the stubs so that one doesn't need to checkout the EE repository to figure out what this function is used for. (It's okay that we duplicate the comments between CE and EE.)

Reply:
Copy pasted comments